README.md

```diff
@@ -62,14 +62,16 @@ After you're finished please remove all the comments and instructions!
 - CSS
 - JavaScript (ES6)
 - Webpack
+-Linters
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[ Add and Remove Books]**
+- **[ Add and Remove list tasks ]**
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
while you did a wonderful job working on the project requirements I noticed you disabled eslint in your index.js file it is agaist mircroverse policy to disable linters because linters help us write clean code. it would be nice if  you enabled eslint. I have attached a screenshot showcasing your work.
![Code base](https://user-images.githubusercontent.com/79506105/208255263-fb33e205-c532-4fe0-8450-f7d0102bf1a4.PNG)
- Secondly I cant add or a remove a task it say in the requirements that The user will be able to mark task completion by selecting the corresponding checkbox. The updated tasks list will be stored in local storage. All the elements of the user interface will be fully functional and the application should be completed at this stage. I suggest you go through the project requirements to fully understand what you are required to do.